src/components/UserMenu/UserMenu.tsx

```diff
@@ -45,14 +53,13 @@ const UserMenu = ({ showPaymentsItem, small = false, onClick, accessModel, curre
 
   return (
     <ul className={styles.menuItems}>
-      {accessModel === 'SVOD' && profilesEnabled && (
+      {accessModel === 'SVOD' && profilesEnabled && selectProfile && (
```

Comment:
what if the `accessModel` is **AVOD** or any other? Should we show the user menu in that case?